### ISSUE TITLE
Error: Invalid dimensions for plot, width = 0, height = 0

### ISSUE THREAD

**Issue by Zauxst** [grafana/grafana#14339]
### What Grafana version are you using?
Grafana v5.4.0 (commit: 69c5191)
### What datasource are you using?
Prometheus
### What OS are you running grafana on?
Debian
### What did you do?
Share Iframe
### What was the expected result?
Iframe Embedded into website
### What happened instead?
Exactly what happens on your website........
http://docs.grafana.org/reference/sharing/#embed-panel

For debug purposes and history reference (in case this issue gets fixed before someone notices this problem) here is the error from your website:
```
Invalid dimensions for plot, width = 622, height = 0

Error: Invalid dimensions for plot, width = 622, height = 0
    at n.resize (https://snapshot.raintank.io/public/build/app.208faab70d3ed01fc138.js:1:227775)
    at new n (https://snapshot.raintank.io/public/build/app.208faab70d3ed01fc138.js:1:197184)
    at https://snapshot.raintank.io/public/build/app.208faab70d3ed01fc138.js:1:223471
    at new a (https://snapshot.raintank.io/public/build/app.208faab70d3ed01fc138.js:1:223632)
    at Function.t.plot (https://snapshot.raintank.io/public/build/app.208faab70d3ed01fc138.js:1:230419)
    at t.callPlot (https://snapshot.raintank.io/public/build/app.208faab70d3ed01fc138.js:22:855106)
    at t.renderPanel (https://snapshot.raintank.io/public/build/app.208faab70d3ed01fc138.js:22:854021)
    at e.<anonymous> (https://snapshot.raintank.io/public/build/app.208faab70d3ed01fc138.js:22:850835)
    at i (https://snapshot.raintank.io/public/build/vendor.208faab70d3ed01fc138.js:165:87589)
    at wa._onCommit (https://snapshot.raintank.io/public/build/vendor.208faab70d3ed01fc138.js:165:89729)
```

**Comment by marefr** [comment]
Duplicate of #14284

**Comment by marefr** [comment]
Fix will be included in Grafana v5.4.1
